client/state/notices/account-recovery/index.js

```diff
@@ -39,7 +39,10 @@ const getUpdateSuccessMessage = ( target, getState ) => {
 			return translate( 'Successfully updated recovery email address.' );
 
 		default:
-			return translate( 'Successfully updated the recovery option.' );
+			return translate( 'Successfully updated the recovery option.', {
+				comment: 'Recovery option here is a generic term for all the ways for account recovery.' +
```

Comment:
Almost! Concatenated strings don't work as arguments for translate() since the parsing that generates the strings file is not actually running JS. You can shorten this up a bit though:
`Recovery option: generic term for methods of account recovery, i.e emails or sms`

To verify your changes, you can run `make translate` and find the string in that `calypo-strings.pot` file.
If the comment has been added, it will look like this:
```
#: client/state/notices/account-recovery/index.js:42
#. Recovery option here is a generic term for all the ways for account recovery. e.g. emails and sms numbers are two ways we are providing at the moment.
msgid "Successfully updated the recovery option."
msgstr ""
```